features/src/python-lit/install.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+set -ex
+
+LIT_VERSION="${VERSION:-latest}";
+
+if [ "$LIT_VERSION" -neq "latest" ]; then
+  LIT_VERSION_TO_INSTALL="lit==$LIT_VERSION"
+else
+  LIT_VERSION_TO_INSTALL="lit"
+fi;
+
+# Ensure we're in this feature's directory during build
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+
+# install global/common scripts
+. ./common/install.sh;
+
+
+check_packages                  \
+    gpg                         \
+    dirmngr                     \
+    apt-utils                   \
+    gettext-base                \
+    bash-completion             \
+    gcc g++ gfortran            \
+    software-properties-common  \
+    ;
+
+apt-get install -y --no-install-recommends python3 python3-dev python3-wheel python3-pip;
+python3 -m pip install wheel;
+python3 -m pip install psutil $LIT_VERSION_TO_INSTALL;
```

Comment:
Features can't currently depend on other features, but I think it's fine to assume python is already installed (e.g. via our `mambaforge` feature, `ghcr.io/devcontainers/features/python`, etc.). I do something similar in the [`rapids-build-utils`](https://github.com/rapidsai/devcontainers/blob/f854296b98ae9feae8147efd5bc2cc91c6361503/features/src/rapids-build-utils/install.sh#L25-L28) feature.

```suggestion
if ! type python >/dev/null 2>&1; then
  echo "lit feature expects python to already be installed" >&2;
  exit 1;
fi
python -m pip install $LIT_VERSION_TO_INSTALL;
```

Reply:
Yeah, I felt weird calling this `python-lit` because I'd assume we won't use `mambaforge` in our matrix.